setup.py

```diff
@@ -51,5 +51,8 @@
         [console_scripts]
         yamlfix=yamlfix.entrypoints.cli:cli
     """,
-    install_requires=["click", "ruyaml"],
+    install_requires=[
+        "click",
+        "ruyaml @ git+git://github.com/pycontribs/ruyaml@0.90.0.2#egg=ruyaml",
```

Comment:
I usually add a comment to point to the open issue that is hardcoding the dependency, otherwise I usually don't remember why is it there.

That way if the issue is solved and I haven't changed it, the next time we stumble with these lines of code, and see it's closed, we will be sure we can unpin the version

Reply:
Good point :smile: I'll add the comment once coveralls is back up and build has a chance of passing :sweat_smile: